tests/test_base.py

```diff
@@ -124,9 +134,15 @@ def _create_product_template(
                 ('create', [{
                     'uri': uri,
                     'displayed_on_eshop': displayed_on_eshop,
+                    'code': code,
                 }])
             ]
-        return self.ProductTemplate.create(vlist)
+        template = self.ProductTemplate.create(vlist)[0]
+        self.file.product = template.products[0]
```

Comment:
this has been replaced with the `product.media`, check nereid-catalog latest code